cache-hazelcast/src/main/java/io/micronaut/cache/hazelcast/HazelcastClientConfiguration.java

```diff
@@ -55,4 +57,28 @@ public class HazelcastClientConfiguration extends ClientConfig {
         super.setNetworkConfig(networkConfig);
         super.getConnectionStrategyConfig().setConnectionRetryConfig(connectionRetryConfig);
     }
+
+    /**
+     * Returns the path to a Hazelcast XML or YAML configuration file.
+     * <p>If non-null, the contents of the file will override this configuration.
+     * This path will be used to set system property {@code hazelcast.client.config}.</p>
+     *
+     * @since 4.1
+     * @return The path to the Hazelcast XML or YAML configuration file.
+     */
+    public String getConfig() {
```

Comment:
Add `@Nullable` to return type.

Reply:
resolved via https://github.com/micronaut-projects/micronaut-cache/pull/671/commits/585f0419f6aaab403462fd738628a6dd6acc807f